spec.md

```diff
@@ -412,6 +412,16 @@ service GroupController {
     }
 }
 
+service SnapshotMetadata {
+  option (alpha_service) = true;
+
+  rpc GetAllocated(GetAllocatedRequest)
```

Comment:
recommend including a noun in the RPC (and related req, resp messages), like we have elsewhere. e.g. `GetBlocksAllocated`, `GetBlocksDelta`, or something like that

Reply:
@PrasadG193 @ihcsim ^

Reply:
`GetBlocksAllocated` and `GetBlocksDelta` sound good to me. In the context of `SnapshotMetadata` service, it's clear that we are talking about blocks metadata, not blocks.

Reply:
`GetBlocksAllocated` and `GetBlocksDelta` sounds a bit misleading to me as we are not returning actual blocks. Maybe `GetBlocksMetadataAllocated`/`GetBlocksMetadataDelta`?

Reply:
I'll go with `GetBlockMetadataAllocated` and `GetBlockMetadataDelta`.